packages/xo-server/src/xapi/index.js

```diff
@@ -1035,7 +1081,7 @@ export default class Xapi extends XapiBase {
         $defer.onFailure(() => this._deleteVdi(newVdi.$ref))
       }
 
-      await asyncMap(vbds[vdiId], vbd =>
+      await asyncMap(vbds[vdiRef], vbd =>
```

Comment:
Does it create a VBD for the `suspend_VDI` as well? If so it shouldn't

Reply:
It's done before the VM creation: https://github.com/vatesfr/xen-orchestra/pull/4252/files#diff-0d7c8183f21c8290ffbeb18bca5c0f88R909-R918